config/locales/en.yml

```diff
@@ -8,6 +8,18 @@ en:
     manage: Manage teacher training applications
     support: Support for Apply
     api: Apply for teacher training API
+    get_into_teaching: Get Into Teaching
+  get_into_teaching:
+    # TODO: Update to 0800 389 2500 when GIT leaves beta
+    tel: 0800 389 2501
+    opening_times: "Monday to Friday, 8.30am to 5pm (except public\u00A0holidays)"
```

Comment:
Why `\00A0` instead of a space?

Reply:
Prevents an orphan appearing in the footer (which is coming in a follow-up PR).

So instead of this:

```
Monday to Friday, 8.30am to 5pm (except public
holidays)
```

we get this:

```
Monday to Friday, 8.30am to 5pm (except
public holidays)
```

Reply:
Nice. Orphan! I haven't heard that expression since subediting class. Skills I never used, obviously